Rambunctious-Reindeers/reactive-nutshell
# Mb styling

# Description

Lots o stylesss


# Testing Instructions

- `git fetch --all`
- `git checkout mb-styling`
- See if friends are styled differently in events + articles
- See if edit and delete functionality is removed if it's not the current user
- See if it be lookin' cute

# Checklist:
